### PR TITLE
Merge v8 updates to master

### PR DESCRIPTION
**Pull request by claireriley**

### CODE REVIEW

File: src/pages/docs/integrations/available-integrations/coralogix.md [hunk 1]
```diff
@@ -31,33 +31,21 @@ Navigate to "Send your logs" tab and copy the private key for later use, as illu
 
 ## Configuring Coralogix Integration
 
-1. In the [Integrations](https://go.postman.co/workspaces) tab for your workspace, select Coralogix from the list of third party integrations.
+1. From the [Integrations](https://go.postman.co/integrations/browse?category=all) page, search and select Coralogix from the list of integrations.
 
-   [![coralogix integrations page](https://assets.postman.com/postman-docs/coralogix_viewdetails.png)](https://assets.postman.com/postman-docs/coralogix_viewdetails.png)
+   ![coralogix integrations page](https://assets.postman.com/postman-docs/coralogix-search-b.jpg)
```
Comment (ch264) [inline src/pages/docs/integrations/available-integrations/coralogix.md:36]: This image is broken

Comment (ch264) [reply src/pages/docs/integrations/available-integrations/coralogix.md:36]: @jerry-li-dev

**Review by ch264 — CHANGES_REQUESTED** [review]

File: src/pages/docs/integrations/available-integrations/coralogix.md (hunk 1 above)
Comment (ch264) [reply src/pages/docs/integrations/available-integrations/coralogix.md:36]: Screen Shot 2021-02-03 at 11 22 08 AM

Comment (jerry-li-dev) [reply src/pages/docs/integrations/available-integrations/coralogix.md:36]: Issue resolved with https://github.com/postmanlabs/postman-docs/pull/2826

File: src/pages/docs/collaborating-in-postman/collaboration-intro.md [hunk 2]
```diff
@@ -82,19 +84,21 @@ All members of your workspace will have avatars displayed at the top of Postman.
 
 ### Creating a new workspace from the menu
 
-Select your current workspace in Postman to open the workspace menu, then click **Create New**.
+Select your current workspace in Postman to open the workspace menu, then click **+ New Workspace**.
+
+[![New workspace](https://assets.postman.com/postman-docs/create-new-workspace-v8.jpg)](https://assets.postman.com/postman-docs/create-new-workspace-v8.jpg)
 
-[![create workspace](https://assets.postman.com/postman-docs/Screen%20Shot%202019-11-11%20at%205.29.42%20PM.png)](https://assets.postman.com/postman-docs/Screen%20Shot%202019-11-11%20at%205.29.42%20PM.png)
+Specify a workspace name and summary. For a team workspace, add collaborators by entering their email addresses, then define their [workspace roles](docs/collaborating-in-postman/roles-and-permissions/#workspace-roles).
```
Comment (ch264) [inline src/pages/docs/collaborating-in-postman/collaboration-intro.md:91]: @jerry-li-dev there is a slash missing here infront of `docs/collaborating-in-postman/roles-and-permissions/#workspace-roles`

Comment (ch264) [inline src/pages/docs/collaborating-in-postman/collaboration-intro.md:91]: @jerry-li-dev slash missing here infront of 'doc/...'

File: src/pages/docs/collaborating-in-postman/using-workspaces/creating-workspaces.md [hunk 3]
```diff
@@ -71,25 +74,35 @@ Click **Create Workspace** and Postman will open your new workspace. You can add
...
 
 [![Enable team profile](https://assets.postman.com/postman-docs/enable-team-profile.jpg)](https://assets.postman.com/postman-docs/enable-team-profile.jpg)
 
 There are four types of workspaces - personal, team, private and public workspaces. When you open a workspace, Postman will provide an overview of its contents, activity, and members in a new tab.
 
+Open the workspace menu, then click **+ New Workspace**.
+
+[![New workspace](https://assets.postman.com/postman-docs/create-new-workspace-v8.jpg)](https://assets.postman.com/postman-docs/create-new-workspace-v8.jpg)
+
+Specify a workspace name and summary. For __Free__ and __Team__ users, to create a public workspace change the workspace visibility to __Public__. Add collaborators by entering their email addresses, then define their [workspace roles](docs/collaborating-in-postman/roles-and-permissions/#workspace-roles).
```
Comment (ch264) [inline src/pages/docs/collaborating-in-postman/using-workspaces/creating-workspaces.md:95]: @jerry-li-dev slach missing infront of 'doc/...'

**Review by ch264 — CHANGES_REQUESTED** [review]

File: src/pages/docs/collaborating-in-postman/collaboration-intro.md (hunk 2 above)
Comment (claireriley) [reply src/pages/docs/collaborating-in-postman/collaboration-intro.md]: @jerry-li-dev I've got this one

File: src/pages/docs/collaborating-in-postman/using-workspaces/creating-workspaces.md (hunk 3 above)
Comment (claireriley) [reply src/pages/docs/collaborating-in-postman/using-workspaces/creating-workspaces.md]: @jerry-li-dev got this one too

**Review by ch264 — APPROVED** [review]
Ran Frog, all clean, LGTM 🚀

**Pull request merged**